src/igvfd/schemas/auxiliary_set.json

```diff
@@ -65,6 +65,16 @@
                 "quantification DNA barcode sequencing",
                 "variant sequencing"
             ],
+            "enum_descriptions": {
+                "cell hashing barcode sequencing": "Sequencing of the antibody barcoding of cell batches.",
+                "cell sorting": "Data collected during fluorescence activated cell sorting.",
+                "circularized RNA barcode detection": "Reporter detection (such as for MPRA) through sequencing circularized barcoded RNA molecules.",
+                "gRNA sequencing": "Sequencing data of guide RNAs for CRISPR screen assays.",
```

Comment:
drop word "screen", (its 90% true but CRISPR is used for things other than screening)

Reply:
fixed